dd-sdk-android-sqldelight/src/main/kotlin/com/datadog/android/sqldelight/DatadogSqliteCallback.kt

```diff
@@ -28,6 +28,7 @@ class DatadogSqliteCallback(schema: SqlDriver.Schema) : AndroidSqliteDriver.Call
         GlobalRum.get()
             .addError(
                 String.format(
+                    Locale.US,
                     DATABASE_CORRUPTION_ERROR_MESSAGE,
                     db.path,
                     Locale.US
```

Comment:
this argument should be removed. Good for us that if number of `varargs` is more than number of placeholders nothing crashes (can be checked here https://ideone.com/LcAHSE). When number of placeholders is more than number of args, it does crash though.